primitives/src/v1/mod.rs

```diff
@@ -892,11 +892,29 @@ pub enum CandidateEvent<H = Hash> {
 #[cfg_attr(feature = "std", derive(PartialEq, Default, MallocSizeOf))]
 pub struct SessionInfo {
 	/// Validators in canonical ordering.
+	///
+	/// NOTE: There might be more authorities in the current session, than validators participating
```

Comment:
It's not clear here whether this contains only the validators participating in parachain consensus or the others as well.